### PR TITLE
add stale-issue project workflow

### PR DESCRIPTION
**Pull request by bobzilladev**
Add some automation, based on: https://docs.github.com/en/actions/managing-issues-and-pull-requests/closing-inactive-issues

### CODE REVIEW

**Comment by salilsub** [comment]
@bobzilladev can we also get a review from @CK-Ward before merging?

File: .github/workflows/stale-issue.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          days-before-issue-stale: 30
+          days-before-issue-close: 14
```
Comment (CK-Ward) [inline .github/workflows/stale-issue.yml:16]: to clarify, the logic here is 30 days to mark as stale. after that designation, it's 14 additional days before the auto close? which is 👍

**Review by CK-Ward — APPROVED** [review]
1 clarifying comment, LGTM overall

File: .github/workflows/stale-issue.yml (hunk 1 above)
Comment (bobzilladev) [reply .github/workflows/stale-issue.yml:16]: aye, label after 30 days of idle, then close 14 days after label.

Comment (CK-Ward) [reply .github/workflows/stale-issue.yml:16]: thanks!

**Pull request merged**